dom.bs

```diff
@@ -2252,6 +2252,10 @@ can be used to explore this matter in more detail.
  {{ProcessingInstruction}}, or {{Comment}}.
 
  <p>Objects that implement {{Text}} sometimes implement {{CDATASection}}.
+
+ <p>Thus, every [=node=]'s [=primary interface=] is one of: {{Attr}}, {{CDATASection}}, {{Comment}},
+ {{Document}}, {{DocumentFragment}}, {{DocumentType}}, {{Element}} or an inherited interface of it,
```

Comment:
```suggestion
 {{Document}}, {{DocumentFragment}}, {{DocumentType}}, or {{Element}}; or an inherited interface of it:
```

Reply:
Although I'm not sure how this is correct. Why is ProcessingInstruction an inherited interface and Comment is not?

Also, why would ShadowRoot not be a primary interface? Isn't that the most derived? And I think both Text and CDATASection are primary interfaces as they are the most derived on the objects they occur on. Just when CDATASection is the primary interface, it's also an inherited interface of Text.

Reply:
The suggested change seems incorrect, and then your further comment seems to be based on confusion about what the spec text would say after that incorrect change.

I'll rephrase the original in another way. Every node's primary interface is one of:

- Attr
- CDATASection
- Comment
- Document
- DocumentFragment
- DocumentType
- Element
- HTMLElement
- HTMLDivElement
- ...
- ProcessingInstruction
- ShadowRoot
- Text

Reply:
I see, that makes sense. Can we put them in the same order as the document outline does? I think I generally try to be consistent about that node order throughout the document.

Reply:
Done!